tests/serverpod_test_server/test_e2e/connection_test.dart

```diff
@@ -732,6 +732,250 @@ void main() {
     });
   });
 
+  group('Database', () {
+    test('Write and read', () async {
+      var dateTime = DateTime.utc(1976, 9, 10, 2, 10);
+      var duration = const Duration(seconds: 1);
+      var uuid = UuidValue.fromString('a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11');
+
+      // TODO: Support ByteData in database store
+      var types = Types(
+        aBool: true,
+        aDouble: 1.5,
+        anInt: 42,
+        aDateTime: dateTime,
+        aDuration: duration,
+        aString: 'Foo',
+        aUuid: uuid,
+        // aByteData: createByteData(),
+      );
+
+      var count = await client.basicDatabase.countTypesRows();
+      expect(count, isNotNull);
+
+      types = await client.basicDatabase.insertTypes(types);
+      expect(types.id, isNotNull);
+
+      var newCount = await client.basicDatabase.countTypesRows();
+      expect(newCount, isNotNull);
+      expect(newCount, equals(count! + 1));
```

Comment:
Suggestion: It seems these tests got added back in.

Reply:
Right. I did not check all the non-conflict merge files, so this slipped passed me. Thanks for pointing it out. The `Types` DB tests are removed from this file again.